Pod/Classes/Pipes.swift

```diff
@@ -52,9 +52,8 @@ Try piping the optional input into the function on the right.
 
 Result is nil if lhs is nil. Otherwise, same as `|>`
 */
-public func |?> <A, B>(lhs: A?, rhs: A -> B) -> B? {
-    return lhs
-        |> map { rhs($0) }
+public func |?> <A, B>(lhs: A?, @noescape rhs: A -> B) -> B? {
+  return lhs.map(rhs)
```

Comment:
Builds with XCode 7.0 (i.e. Swift 2.0).
Build fails with XCode 6.4.
Which versions do you want to support? Is there a way to let Travis check with older XCode versions?

For maximum compatibility we should probably implement the `|?>` operator manually:
```
public func |?> <A, B>(lhs: A?, @noescape rhs: A -> B) -> B? {
  if let a = lhs {
    return rhs(a)
  } else {
    return nil
  }
}
```